README.md

```diff
@@ -1 +0,0 @@
-# learn-react
```

Comment:
Need to make sure this file isn't affected by your PR.

Right now your git has tracked this file as being deleted, so it would delete it if I merged your PR.

If you add this file back in your local, and push it back up, we should be good to go